README.md

```diff
@@ -60,69 +59,85 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 samburu tech-week website <a name="about-project"></a>
 
+**samburu tech-week website** is a javascript, css and html project undertaken as end of module1 capstone.
 ## 🛠 Built With <a name="built-with"></a>
 
+- Major languages javaScript HTML and CSS
+- Frameworks -none
+- Technologies used -Linters
+ 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="#">HTML</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="#">None</a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="#">None</a></li>
   </ul>
 </details>
 
+<!-- Features -->
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Mobile Menu**
+- **Responve website**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [see Live](https://lengushuru.github.io/samburu-tech-week/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 project presentation <a name="live-demo"></a>
+
+- [presentation](https://www.loom.com/share/ae31e39ca92947f8a6b9fb31f5a5bc08)
```

Comment:
- [ ] Great effort recording the loom video showing us what you have in your project, 👍 👍 but I notice the project you presented in the video looks a little bit different from the one you have now. the `nav logo` and the `nav link` of the project you have in the video presentation look slightly different from the one you have now in your project.

  - Also kindly show some pieces of code you used in the project that you are most proud of. I notice in your presentation you only take us through the project on the browser alone and you didn't show us your codebase.